Assets/MixedRealityToolkit/Inspectors/Setup/BuildDeployWindow.cs

```diff
@@ -135,7 +123,7 @@ private static bool CanInstall
                 bool canInstall = true;
                 if (EditorUserBuildSettings.wsaSubtarget == WSASubtarget.HoloLens)
                 {
-                    canInstall = DevicePortalConnectionEnabled;
+                    canInstall |= DevicePortalConnectionEnabled && IsHoloLensConnectedUsb;
```

Comment:
With `|=`, I'm pretty sure this will always return `true`, since `canInstall` is defaulted to `true` above. Seems reasonable to leave it as `=`.

```suggestion
                    canInstall = DevicePortalConnectionEnabled && IsHoloLensConnectedUsb;
```